syntax/parser.go

```diff
@@ -43,8 +43,8 @@ expr   -> C* amp="&"* @ C* arrow=(
         > C* "{" C* rel=(names tuple=("(" v=@:",", ")"):",",?) "}" C*
         | C* "{" C* set=(elt=@:",",?) "}" C*
         | C* "{" C* dict=((key=@ ":" value=@):",",?) "}" C*
-        | C* cond=("cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr)? ")") C*
-        | C* cond=(("(" control_var=expr ")" | IDENT)? C* "cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr)? ")") C*
+        | C* cond=("cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr)? ","? ")") C*
+        | C* cond=(("(" control_var=expr ")" | IDENT)? C* "cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr)? ","? ")") C*
```

Comment:
This is an autogenerated file, we have the file `arrai.wbnf` now, make your grammar changes there and then run `make`. It will update this file

Reply:
got it, thanks Novan.